docs/source/algorithms/BilbySANSDataProcessor-v1.rst

```diff
@@ -0,0 +1,64 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+The main equation Bilby is using for the :math:`I(Q)` calculation is the following:
+
+.. math:: I(Q)=\frac{1}{d_{sam}}.\frac{\sum_{R, \lambda \subset Q}C_{sam,corr}(R,\lambda)}{M \sum_{R,\lambda \subset Q}T_{corr}(R, \lambda)\frac{I_{empty\_beam}(\lambda)}{M_{empty\_beam}att_{empty\_beam}}\Omega(R)Det_{flood}(R)}
+
+Where :math:`C_{sam, corr}` is the measured counts per pixel per wavelength,
+:math:`I_{empty\_beam}` is the intensity of the empty beam collected for :math:`M_{empty\_beam}` time,
+M is a time measure for the data collection, :math:`att_{empty\_beam}` is attenuation factor,
+:math:`\Omega(R)` is solid angle, :math:`Det_{flood}` is a detector response function,
+:math:`T_{corr}` is the sample transmission, and :math:`d_{sam}` is the sample thickness.
+
+Details are described in the paper
+[Sokolova, A., Christoforidis, J., Eltobaji, A., Barnes, J., Darmann, F., Whitten, A. E. & de Campo, L. (2016). Neutron News 27, 9-13]
```

Comment:
This could rather be a reference to the citation provided below: `Details are described in a paper [#Sokolova]_` or similar.